Bottom: ddfc4411dfb1d75e67c2158cab013c8881522942
Top:    9b0b412eb6512b8a6a62d02b83d63db385b53214
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-28 02:30:00 +0100

Refresh of test-remove-some-more-gdk-stuf

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 50ac011..e3d305c 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -912,7 +912,6 @@ gboolean
 ghid_pinout_preview_expose (GtkWidget *widget,
                             GdkEventExpose *ev)
 {
-  extern HID ghid_hid;
   GhidPinoutPreview *pinout = GHID_PINOUT_PREVIEW (widget);
   GdkDrawable *save_drawable;
   double save_zoom;
@@ -1007,7 +1006,6 @@ ghid_pinout_preview_expose (GtkWidget *widget,
 GdkPixmap *
 ghid_render_pixmap (int cx, int cy, double zoom, int width, int height, int depth)
 {
-  extern HID ghid_hid;
   GdkPixmap *pixmap;
   GdkDrawable *save_drawable;
   double save_zoom;
